nextcloud_update.sh

```diff
@@ -591,6 +591,18 @@ $DOCKER_RUN_OUTPUT"
     docker_update_specific 'plex' "Plex Media Server"
 fi
 
+# Fix Collabora change too coolwsd
+if grep -r loolwsd $SITES_AVAILABLE/*
+then
```

Comment:
maybe add a loop here if there are more than one, shouldn't be, but you never know.